code-challenges.js

```diff
@@ -14,29 +14,164 @@ const fruit2 = "banana"
 const fruit3 = "cherry"
 const fruit4 = "kiwi"
 
+// Function Signature
+// input: words
+// output: string indicating the word with the most characters
+
+// Example
+// input: "apple", "banana" // output --> "banana"
+// input: "cherry", "kiwi" // output --> "cherry"
+
+// Process
+// create a function that will take in two words and count the characters in each word and only return the word with the most characters
+// two fruits were named in the beginning of the function
+// next the lengths of each fruit word are established by using .length
+// we asked for a return if the first fruit name was longer than the second and if not then return the name of the second fruit
+
+const longerName = (fruit1, fruit2) => {
+let fruit1length = fruit1.length
+let fruit2length = fruit2.length
+if(fruit1length > fruit2length) {
+    return fruit1
+} else {
+    return fruit2
+}
+}
+console.log(longerName("apple", "banana"))
+console.log(longerName("cherry", "kiwi"))
 
 // --------------------1) Create a function that takes in any given number and determines if the number is below boiling point, at boiling point, or above boiling point. Boiling point is 212 degrees Fahrenheit. Use the test variables provided below. Expected output: "42 is below boiling point", "350 is above boiling point", "212 is at boiling point"
 
 const temp1 = 42
 const temp2 = 350
 const temp3 = 212
 
+// Function Signature
+// input: numbers
+// output: "42 is below boiling point", "350 is above boiling point", "212 is at boiling point"
+
+// Example
+// input: 42 // output --> "42 is below boiling point"
+// input: 350 // output --> "350 is above boiling point"
+// input: 212 // output --> "212 is at boiling point"
+
+// Process
+// 
+
+const boilingPoint = (temp1, temp2, temp3) => {
+if(temp1 < 212) {
+    console.log("42 is below boiling point")
+} else if(temp2 > 212) {
+    console.log("350 is above boiling point")
+} else if(temp3 = 212) {
+    console.log("212 is at boiling point")
+}
+}
+console.log(boilingPoint(42))
+console.log(boilingPoint(350))
+console.log(boilingPoint(212))
```

Comment:
Alright I get ya! First Tip: Pseudocode is the art of writing out all your thoughts about the code and creating documentation for yourself to interact with. 

If you write out the line " I want this code to work with any temperature" We already start moving along the current issue of hard coding our returns. 

second the assignment operator and the equality operator are different things in Javascript. I harp on this a lot in class for the exact issue you ran into on line 66.  this would have been helped with writing out the thought "the temperature needs to be compared to 212 exactly. What's the operator for that?" 

On line 70-72 you are correctly calling your function and passing it one argument, good job! But on 61 you have set up your function to take 3 parameters. This could have been solved by pseudocoding out "boilingPoint takes in a parameter of a number, but for this function I will use temperature"  or something like that. 

Good attempt! I think all of the code issues you are going to run into for the next couple weeks will be solvable with good pseudo code. Get on it!